### ISSUE TITLE
use new commit but got bad mesh

### ISSUE THREAD

**Issue by ansj11** [Anttwo/SuGaR#197]
I used old commit 87d8594d0a8938b5bdcf722d489a0d2642b384e5 and got a complete mesh. But got a incomplete mesh when I upgraded to commit new, why are they so different?

<img width="596" alt="截屏2024-06-06 11 43 44" src="https://github.com/Anttwo/SuGaR/assets/24786133/3cc6585a-94d8-4d85-8e87-811c58794480">
<img width="636" alt="截屏2024-06-06 11 43 58" src="https://github.com/Anttwo/SuGaR/assets/24786133/47f53820-c31d-4657-a7fa-648ab3ab8209">

https://github.com/Anttwo/SuGaR/assets/24786133/aa958222-7192-4436-a2d4-99cc11cf2f66

**Comment by kusstox** [comment]
I experience the same issue, have you managed to fix it?

**Comment by ansj11** [comment]
> I experience the same issue, have you managed to fix it?

no, I didn't use this method. Now I use 2DGS[!https://github.com/hbb1/2d-gaussian-splatting.git]

**Comment by Anttwo** [comment]
Hi all,
I'm really sorry for not answering sooner, I'm swamped under work these days!
Yeah, 2DGS is really really awesome :D

I will push a Blender Addon for SuGaR in the coming days though, so you might still be interested in the SuGaR method if you want to edit or animate your Gaussian Splatting+Mesh without a single line of code!

I have a question concerning the issue you had:
Are you using now the `white_background` parameter?
I'm thinking, since your car is actually white, I think the optimization might just use the background to reconstruct the white parts, which might explain why your mesh only contains the black parts.

Have you tried rerunning the code with black background instead (just like the previous commit)?
